### PR TITLE
Improved Things 3 To Do Plugins + Housekeeping

### PR DESCRIPTION
**Pull request by mcclowes**
- Moved Things 3 plugins from `Lifestyle` -> `Lifestyle/Things`
- Moved To Do plugins from `Lifestyle` -> `Lifestyle/ToDo`
- Renamed Things 3 plugin to Things - View Today, Lifestyle/things3.10s.sh → Lifestyle/Things/thingstoday.6s.sh
- Improved Things - View Today - Filters projects, shows completion state
- Added Things - View Inbox - Show users Inbox

### CODE REVIEW

**Comment by mcclowes** [comment]
Ah, so I'm actually trying to achieve word splitting there. I'm not really sure how to achieve this functionality another way...

**Comment by matryer** [comment]
@mcclowes please fix conflicts and I'll merge this in today.

**Comment by gingerbeardman** [comment]
Please solve the outstanding Travis CI issue(s) so this can be merged! Many thanks.

You can close and reopen the PR to get Travis to try again, see #1214

**Comment by tresni** [comment]
Sorry for the necro post, I'll try to get this merged (finally),

TL;dr - to disable you;d simply add a new line of `# shellcheck disable=SC2086` on the line preceding the offending line.
